lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -898,11 +989,44 @@ public struct SwiftifyImportMacro: PeerMacro {
   }
 
   static func setNonescapingPointers(_ args: inout [ParamInfo], _ nonescapingPointers: Set<Int>) {
+    if args.isEmpty {
+      return
+    }
     for i in 0...args.count - 1 where nonescapingPointers.contains(paramOrReturnIndex(args[i].pointerIndex)) {
       args[i].nonescaping = true
     }
   }
 
+  static func setLifetimeDependencies(_ args: inout [ParamInfo], _ lifetimeDependencies: [SwiftifyExpr: [LifetimeDependence]]) {
+    if args.isEmpty {
+      return
+    }
+    for i in 0...args.count - 1 where lifetimeDependencies.keys.contains(args[i].pointerIndex) {
+      args[i].dependencies = lifetimeDependencies[args[i].pointerIndex]!
+    }
+  }
+
+  static func lifetimeAttributes(_ funcDecl: FunctionDeclSyntax,
+    _ dependencies: [SwiftifyExpr: [LifetimeDependence]]) -> [AttributeListSyntax.Element] {
+    var result : [AttributeListSyntax.Element] = []
+    for dependence in dependencies[.`return`, default: []] {
```

Comment:
What happens if there are multiple dependencies listed? Is it okay to have multiple `@lifetime` attributes? What happens if one is borrowing and one is copying? We should have tests covering these cases.

Reply:
Good catch! Actually, it looks like I misunderstood  this attribute on the Swift side. It looks like we need to have a single attributes where the multiple dependencies are listed as a comma separated list. Updated the code and added the requested tests.